NuWeather/plugin.py

```diff
@@ -377,7 +410,7 @@ def _darksky_fetcher(self, location):
         # N.B. Dark Sky docs tell to not expect any values to exist except the timestamp attached to the response
         return {
             'location': display_name,
-            'poweredby': 'Dark\xa0Sky+' + self._geocode.backend,
+            'poweredby': 'DarkSky+' + geocode_backend,
             'url': 'https://darksky.net/forecast/%s,%s' % (lat, lon),
```

Comment:
(One last thing) Can we keep the proper branding with the space? Same for "Google Maps" instead of GoogleMaps. The \xa0 char is a non breaking space to prevent splitting the name onto separate lines.

Anyways I'll try to test this soon, and probably add a more convenient way to clear the geocode DB so switches actually apply